.github/workflows/canary.yml

```diff
@@ -0,0 +1,99 @@
+name: Publish canary on PR comment
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  check_auth:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/canary' }}
+    steps:
+      - name: Check auth
+        id: check_auth
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          github-token: ${{ secrets.GH_TOKEN }}
+          script: |
+            const { owner, repo } = context.issue;
+            const { login: username } = context.payload.comment.user;
+            await github.rest.repos.checkCollaborator({
```

Comment:
Thoughts on requiring > collaborator permissions to request a canary build? Should we just let anyone do it?

Reply:
I think requiring collab is the right call -- it allows us to double check stuff before running a build w/ elevated permissions and also potentially save on build resources

Reply:
I would say everyone shouldn't be able to do it right now (maybe something we can reevaluate in the future though) since right now we entitle can feasibly keep up with giving people those permissions.